### ISSUE TITLE
Store inventory download not working (Downloading 0KB)

### ISSUE THREAD

**Issue by AllesMeins** [rgriebl/brickstore#124]
Hi,

I've an issue with downloading my store inventory. The "Downloading window" is displayed indefinitly with a full progress bar saying Download: 0KB. This has been going on since I started using BrickStore at the end of December, so I can rule out a temporary server failure.
Downloading the XML with my browser and importing works fine.

I'm on Ubuntu 20.04 and I'm using the most recent version 2021.1.6

**Comment by rgriebl** [comment]
Works fine here. Did you enter your Bricklink credentials correctly in the Settings dialog?

**Comment by AllesMeins** [comment]
Yes, checked it multiple times. Is there some kind of debug log I could check to see what might be going on?

**Comment by paramecie** [comment]
Do you have "exotic" chars in your password?
Maybe try (TEMPORARILY) set up a simple password (only chars and numbers) on BrickLink like "abcde12345" and do the same on BrickStore and see if now it works (and then revert to your usual password)?

**Comment by rgriebl** [comment]
I tried to use a wrong username / password, but this only results in an XML parse error for me (that's to be expected). Timing out is normally a sign of a firewall silently dropping the connection without telling the application.
I don't have an Ubuntu VM here at the moment to test it, but my Debian/sid builds work perfectly fine.
You can have a look at the error log -- see the View menu. Maybe something does come up there.

**Comment by rgriebl** [comment]
Just tested it on a fresh 20.04, fully updated: works fine.
(installed via `sudo apt install ./Ubuntu-20.04-brickstore_2021.1.6_amd64.deb`)

**Comment by AllesMeins** [comment]
Thank you for your effort and sorry for getting back to you so late. I finally figured it out and I'm afraid it was my own stupidity :-( I've set "Username" to my mailadress because this is what I use to login into bricklink. I didn't even remember that I also have a "real" username which can be used to login. After putting in the username everything works as expected. Sorry for wasting your time :-(
